--- gizmod-3.5/libH/Average.hpp~	2007-11-07 12:22:02.000000000 +0000
+++ gizmod-3.5/libH/Average.hpp	2012-06-09 21:21:34.713407261 +0000
@@ -36,6 +36,7 @@
 #endif
 
 #include <vector>
+#include <stdlib.h>
 
 //////////////////////////////////////////////////////////////////////////////
 // Namespace
--- gizmod-3.5/libH/DynamicBuffer.hpp~	2007-11-07 12:22:02.000000000 +0000
+++ gizmod-3.5/libH/DynamicBuffer.hpp	2012-06-09 21:21:57.903406336 +0000
@@ -34,6 +34,7 @@
 #endif
 
 #include <vector>
+#include <stdlib.h>
 
 //////////////////////////////////////////////////////////////////////////////
 // Namespace
